src/popup/utils/bigint.ts

```diff
@@ -0,0 +1,7 @@
+import JsonBig from 'json-bigint';
+
+const jsonBig = JsonBig({ storeAsString: true });
+
+export function parseBigIntJson(response: any): any {
+  return jsonBig.parse(jsonBig.stringify(response));
```

Comment:
Stringifying object and then parsing it again can be unpredictable. I strongly suggest not to use this approach to apply data manipulation.

Reply:
This is how it is often done for the BigInt responses. I see this as totally valid.  If not for this then the engine in general doesn't know how to serialize/deserialize JSON objects with BigInts.